compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -161,6 +161,11 @@ const char *getIreeArchNameForTargetTriple(llvm::Triple triple) {
   return "unknown";
 }
 
+bool isLLVMCPUBackend(IREE::HAL::ExecutableTargetAttr targetAttr) {
+  return targetAttr &&
+         targetAttr.getBackend().getValue().starts_with("llvm-cpu");
```

Comment:
Can't you just check equality ? Why starts_with? If that was copying the vmvx case below, I might have done that out of ignorance in the past.

Reply:
Oh, yes. Thanks for pointing it out! I'm mainly following the other cases, but can definitely check equality.